pkg/runner/runner_test.go

```diff
@@ -57,6 +57,7 @@ func runTestJobFile(ctx context.Context, t *testing.T, tjfi TestJobFileInfo) {
 		assert.Nil(t, err, workdir)
 		fullWorkflowPath := filepath.Join(workdir, tjfi.workflowPath)
 		runnerConfig := &Config{
+			ForceRebuild:          true,
```

Comment:
This should be default false to preserve current behavior.

Reply:
Will correct that